examples/tutorials/physics_unit_tests.py

```diff
@@ -0,0 +1,312 @@
+# [setup]
+import math
+import os
+import time
+
+import magnum as mn
+import matplotlib.pyplot as plt
+import numpy as np
+
+import habitat_sim
+from habitat_sim.utils import viz_utils as vut
+
+dir_path = os.path.dirname(os.path.realpath(__file__))
+data_path = os.path.join(dir_path, "../../data")
+output_path = os.path.join(dir_path, "physics_unit_tests_output/")
+
+save_index = 0
+
+
+# Define Helper Functions (Taken from rigid_object_tutorial.py)
+
+
+def remove_all_objects(sim):
+    for id_ in sim.get_existing_object_ids():
+        sim.remove_object(id_)
+
+
+def place_agent(sim, position, orientation):
+    # place our agent in the scene
+    agent_state = habitat_sim.AgentState()
```

Comment:
position and state, can and should bet set from the constructor if they are known ahead of time.